Omeprazole/Omeprazole-Report.md

```diff
@@ -0,0 +1,625 @@
+
+**Elimination**: Almost 80% of an oral dose of omeprazole is excreted as metabolites in the urine, the remainder in the faeces, primarily originating from bile secretion.
+
+**Linearity/non-linearity:** The AUC of omeprazole increases with repeated administration following a non-linear dose-AUC relationship. This time- and dose- dependency is due to a decrease of first pass metabolism and systemic clearance probably caused by an inhibition of the CYP2C19 enzyme by omeprazole and/or its metabolites (e.g. the sulphone).
+
+**Stereoselectivity:** Because both enantiomers are substrates and inhibitors of CYP2C19, the PK profile of each enantiomer is expected to differ when the racemic mixture is administered as compared to as a single agent because of mutual inhibition between the enantiomers ([Äbelö 2000](#5-references)). The Ki for R-omeprazole is 2-5 higher compared to S-omeprazole ([Liu 2005](#5-references), [Wu 2014](#5-references)).
+
+# 2 Methods
+
+
+## 2.1 Modeling Strategy
+The general workflow for building an adult PBPK model has been described by Kuepfer et al. ([Kuepfer 2016](#5-references)). Relevant information on the anthropometry (height, weight) was gathered from the respective clinical study, if reported. Information on physiological parameters (e.g. blood flows, organ volumes, hematocrit) in adults was gathered from the literature and has been incorporated in PK-Sim® as described previously ([Willmann 2007](#5-references)). The  applied activity and variability of plasma proteins and active processes that are integrated into PK-Sim® are described in the publicly available 'PK-Sim® Ontogeny Database Version 7.3' ([PK-Sim Ontogeny Database Version 7.3](#5-references)).
+
+The model includes distinct molecules for S- and R-omeprazole, the racemic omeprazole is represented by an observer that sums up the concentrations of S- and R-omeprazole. It was assumed that both isomers correspond to the half of the racemic omeprazole dose.
+
+In general, the following step-wise workflow was followed:
+
+1a Define distribution and metabolism for S-omeprazole
+
+1b.Mechanism based inactivation of CYP2C19 by omeprazole
+
+2.Define S-omeprazole absorption based on p.o. 
+
+3.Capsule formulation 
+
+- 3a. Building racemic omeprazole
+
+- 3b. Adjust CYP2C19 expression in gut
+
+4.Define metabolism for R-omeprazole
+
+5.Refine CYP3A4 metabolism on CYP2C19 PM data
+
+6.Refine CYP2C19 metabolism on CYP2C19 EM data
```

Comment:
Formatting: 1a without dot after it, insert blanks between . after the number and the caption
(1b.Mechanism based... => 1b. Mechanism based... etc.)